### PR TITLE
[v10] Backport PagerDuty edits

### LINKED ISSUE #14576: Refresh the Pagerduty access requests guide
## Details
See #14236

### Category
- Improve Existing

### PR DESCRIPTION
**Pull request by ptgott**
Backports #15498
Backports #13729

- Update the PagerDuty plugin guide and add context
- Document the PagerDuty plugin Helm chart

**Reviewer note:** This was a pretty conflict-ridden backport, so please make sure nothing looks off. Thanks!

### CODE REVIEW

File: docs/pages/includes/plugins/identity-export.mdx [hunk 1]
```diff
@@ -52,7 +52,34 @@ This command should result in three PEM-encoded files: `auth.crt`,
...
+$ tctl auth sign --user=access-plugin --out=auth
+```
+
+Then create a Kubernetes secret:
+
+```code
+$ kubectl create secret generic teleport-mattermost-identity --from-file=auth_id=auth.pem
+```
 
+Teleport's Access Request plugins listen for new and updated Access Requests by
```
Comment (alexfornuto) [inline docs/pages/includes/plugins/identity-export.mdx:71]: Should "Access Requests" be capitalized? Does this refer to ephemeral requests or to a proper noun like the plugins? 

```suggestion
Teleport's Access Request plugins listen for new and updated access requests by
```

File: docs/pages/reference/helm-reference/teleport-plugin-pagerduty.mdx [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+<Tabs>
+  <TabItem label="values.yaml">
+  ```yaml
+  teleport:
+    address: "teleport.example.com:3025"
+  ```
+  </TabItem>
+  <TabItem label="--set">
+  ```code
+  $ --set teleport.address="teleport.example.com:3025"
```
Comment (alexfornuto) [inline docs/pages/reference/helm-reference/teleport-plugin-pagerduty.mdx:31]: Is there a reason not to include the binary one would run with this flags in the example?

File: docs/pages/reference/helm-reference/teleport-plugin-pagerduty.mdx [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+  </TabItem>
+</Tabs>
+
+## `pagerduty.url`
+
+| Type | Default value | Required? |
+| - | - | - |
+| `string` | `""` | Yes |
+
+Base URL of the PagerDuty instance
```
Comment (alexfornuto) [inline docs/pages/reference/helm-reference/teleport-plugin-pagerduty.mdx:100]:
```suggestion
Base URL of the PagerDuty instance.
```

File: docs/pages/includes/plugins/identity-export.mdx (hunk 1 above)
Comment (ptgott) [reply docs/pages/includes/plugins/identity-export.mdx:71]: This is a Teleport product, so I've been asking for it to be capitalized.

File: docs/pages/reference/helm-reference/teleport-plugin-pagerduty.mdx (hunk 2 above)
Comment (ptgott) [reply docs/pages/reference/helm-reference/teleport-plugin-pagerduty.mdx:31]: This is the convention we've included in all of our Helm references.

**Review by alexfornuto — APPROVED** [review]

**Review by zmb3 — APPROVED** [review]

**Pull request merged**